frontend/src/components/common/LoginBtn/LoginModal/LoginModal.jsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from 'react';
+import Modals from '../../Modals/Modals';
+import styles from './LoginModal.module.scss';
+import IconButton from '../../../ui-kit/components/IconButton';
+import { ICONS } from '../../../ui-kit/icons';
+import Button from '../../../ui-kit/components/Button';
+import ResetPassword from '../ConfirmationModals/ResetPassword';
+import GoogleLogin from '../GoogleLogin';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+const LoginModal = ({ toggleLogin, toggleRegister }) => {
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [submitted, setSubmitted] = useState(false);
+  const [resetPassword, setResetPassword] = useState(false);
+  const [resetPasswordClicked, setResetPasswordClicked] = useState(false);
+  const [loginSuccess, setLoginSuccess] = useState(false);
+  const [emailError, setEmailError] = useState('');
```

Comment:
Do we need this? you just set it to ' ' but never to real value

Reply:
yes, we need it